### PR TITLE
Update wg-policy

### PR DESCRIPTION
**Pull request by hannibalhuang**
Update wg-policy description with 2018 Fall Cycle latest arrangements

### CODE REVIEW

**Comment by jdumars** [comment]
Is there documentation somewhere for this change?

File: wg-policy/README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
...
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1ihFfEfgViKlUMbY2NKxaJzBkgHh-Phk5hqKTzK-NEEs/edit?usp=sharing).
 
 ## Organizers
 
 * Howard Huang (**[@hannibalhuang](https://github.com/hannibalhuang)**), Huawei
 * Torin Sandall (**[@tsandall](https://github.com/tsandall)**), Styra
-* David Oppenheimer (**[@davidopp](https://github.com/davidopp)**), Google
-* Clayton Coleman (**[@smarterclayton](https://github.com/smarterclayton)**), Red Hat
-* Xuan Jia (**[@xuanjia](https://github.com/xuanjia)**), China Mobile
+* Yisui Hu (**[@easyway](https://github.com/easyway)**), Google
```
Comment (easeway) [inline wg-policy/README.md]: It's "easeway", not "easyway".

**Comment by hannibalhuang** [comment]
@easeway sorry dude just nixed it lol

**Comment by hannibalhuang** [comment]
@jdumars Hi plz refer to https://docs.google.com/document/d/1ihFfEfgViKlUMbY2NKxaJzBkgHh-Phk5hqKTzK-NEEs/edit#heading=h.gzt6i8kf2a0h on the May 17th meeting notes :)

File: sigs.yaml [hunk 2]
```diff
@@ -1967,19 +1967,19 @@ workinggroups:
...
         github: tsandall
         company: Styra
-      - name: David Oppenheimer
-        github: davidopp
+      - name: Yisui Hu
+        github: easeway
         company: Google
-      - name: Clayton Coleman
-        github: smarterclayton
+      - name: Erica Von Beulow
```
Comment (ericavonb) [inline sigs.yaml]: `Erica von Buelow` (lowercase 'v', and 'u' before 'e') 🙂

File: wg-policy/README.md [hunk 3]
```diff
@@ -11,16 +11,16 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
...
 
 ## Organizers
 
 * Howard Huang (**[@hannibalhuang](https://github.com/hannibalhuang)**), Huawei
 * Torin Sandall (**[@tsandall](https://github.com/tsandall)**), Styra
-* David Oppenheimer (**[@davidopp](https://github.com/davidopp)**), Google
-* Clayton Coleman (**[@smarterclayton](https://github.com/smarterclayton)**), Red Hat
-* Xuan Jia (**[@xuanjia](https://github.com/xuanjia)**), China Mobile
+* Yisui Hu (**[@easeway](https://github.com/easeway)**), Google
+* Erica Von Beulow (**[@ericavonb](https://github.com/ericavonb)**), Red Hat
```
Comment (ericavonb) [inline wg-policy/README.md]: Same here - `Erica von Buelow`

File: sigs.yaml (hunk 2 above)
Comment (hannibalhuang) [reply sigs.yaml]: nixed it :)

File: wg-policy/README.md (hunk 3 above)
Comment (hannibalhuang) [reply wg-policy/README.md]: this is an auto generated file, so should be fixed as well :)

**Comment by hannibalhuang** [comment]
could someone help land this patch , there is also another issue that is related to this one #2546

**Pull request merged**